tabular/src/autogluon/tabular/models/lgb/lgb_utils.py

```diff
@@ -23,6 +23,9 @@
         mean_squared_error='l2',
         root_mean_squared_error='rmse',
     ),
+    QUANTILE: dict(
+        pinball_loss='quantile',
```

Comment:
We should map every autogluon-supported quantile metric to 'quantile' right?
I think you'll want to maintain a QUANTILE_METRICS constant list (which for now can just be = [ 'pinball_loss']) that eventually will contain all supported metrics.

Reply:
lgb model is removed.